platform/jvm/gradle/libs.versions.toml

```diff
@@ -1,18 +1,18 @@
 [versions]
 androidBenchkmarkPlugin = "1.2.4"
-androidGradlePlugin = "8.2.2" # have to pin to this version due to 8.3.x breaking our lint https://issuetracker.google.com/issues/332755363
+androidGradlePlugin = "8.8.0" # have to pin to this version due to 8.3.x breaking our lint https://issuetracker.google.com/issues/332755363
```

Comment:
nice they literally released the 8.8 stable version with the fix we needed last week